share/ci/install/pypicongpu.sh

```diff
@@ -43,8 +43,8 @@ micromamba activate pypicongpu
 python3 --version
 MODIFIED_REQUIREMENT_TXT_PICMI=$CI_PROJECT_DIR/lib/python/picongpu/picmi/modified_requirements.txt
 MODIFIED_REQUIREMENT_TXT_PYPICONGPU=$CI_PROJECT_DIR/lib/python/picongpu/pypicongpu/modified_requirements.txt
-python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py $CI_PROJECT_DIR/lib/python/picongpu/picmi/requirements.txt $MODIFIED_REQUIREMENT_TXT_PICMI
-python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py $CI_PROJECT_DIR/lib/python/picongpu/pypicongpu/requirements.txt $MODIFIED_REQUIREMENT_TXT_PYPICONGPU
+python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py PYPIC_DEP_VERSION_referencing PYPIC_DEP_VERSION_jsonschema $CI_PROJECT_DIR/lib/python/picongpu/picmi/requirements.txt $MODIFIED_REQUIREMENT_TXT_PICMI
```

Comment:
Just for my information: Where do these environment variables come from?

Reply:
The script `pypicongpu_generator.py` generates the jobs dynamically during the CI pipeline lifetime. It set also the environment variables.